File: app/src/main/java/com/overswayit/plesnisavezsrbije/models/EmailClubContact.java
package com.overswayit.plesnisavezsrbije.models;

import android.util.Pair;

import com.overswayit.plesnisavezsrbije.R;

import java.util.ArrayList;

/**
 * Created by lazarristic on 18/04/2019.
 * Copyright (c) 2019 PlesniSavezSrbije. All rights reserved.
 */
public class EmailClubContact extends ClubContact {

    public EmailClubContact(String email) {
        super(email);

        Pair<String, Integer> pair = new Pair<>(super.getContact(), R.drawable.ic_email);
        ArrayList<Pair<String, Integer>> arrayList = new ArrayList<>();
        arrayList.add(pair);

        super.setContacts(arrayList);
        super.setType(ClubContactType.EMAIL);
    }
}

File: app/src/main/java/com/overswayit/plesnisavezsrbije/models/EmailClubContact.kt
package com.overswayit.plesnisavezsrbije.models

import android.util.Pair

import com.overswayit.plesnisavezsrbije.R

import java.util.ArrayList

/**
 * Created by lazarristic on 18/04/2019.
 * Copyright (c) 2019 PlesniSavezSrbije. All rights reserved.
 */
class EmailClubContact(email: String?) : ClubContact(email) {

    init {
        val pair = Pair<String?, Int>(super.contact, R.drawable.ic_email)
        val arrayList = ArrayList<Pair<String?, Int>>()
        arrayList.add(pair)

        super.contacts = arrayList
        super.type = ClubContactType.EMAIL
    }
}
